src/output/mod.rs

```diff
@@ -376,17 +303,17 @@ pub fn json(config: Config, ratelimit: Option<&Ratelimiter>) {
                 miss: response_miss,
             };
 
-            let json = JsonSnapshot {
+            let json = RpcPerfSnapshot {
                 window: window_id,
                 elapsed,
                 target_qps: ratelimit.as_ref().map(|ratelimit| ratelimit.rate()),
-                client: Client {
+                client: ClientStats {
                     connections,
                     requests,
                     responses,
                     request_latency: heatmap_to_buckets(&REQUEST_LATENCY),
                 },
-                pubsub: Pubsub {
+                pubsub: PubSub {
```

Comment:
Curious why this is `PubSub` instead of `PubSubStats` to match the `ClientStats` above.

Reply:
Yeah, I was going for the min-change from the existing names (just enough to avoid ambiguity), but making them consistent is probably better.